frontend/src/main/components/BasicCourseSearch/BasicCourseSearchForm.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from "react";
+import { Form, Button, Container, Row, Col } from "react-bootstrap";
+import useSWR from "swr";
+import { toast } from "react-toastify";
+
+import { allTheSubjects } from "fixtures/subjectFixtures";
+import { fetchSubjectAreas } from "main/services/subjectAreaService";
+import { quarterRange } from "main/utils/quarterUtilities";
+
+import SingleQuarterDropdown from "../Quarters/SingleQuarterDropdown";
+import SingleSubjectDropdown from "../Subjects/SingleSubjectDropdown"
+import SingleLevelDropdown from "../Levels/SingleLevelDropdown"
+
+const BasicCourseSearchForm = ({ setCourseJSON, fetchJSON }) => {
+	const quarters = quarterRange("20084", "20222");
+	const levels = [["L","Undergrad-Lower"], 
+					["S","Undergrad-Upper Division"], 
+					["U","Undergrad-All"], 
+					["G","Graduate"]];
+
+    const localSubject = localStorage.getItem("BasicSearch.Subject");
+    const localQuarter = localStorage.getItem("BasicSearch.Quarter");
+	const localLevel = localStorage.getItem("BasicSearch.CourseLevel");
+	
+	const firstDepartment = allTheSubjects[0].subjectCode;
+	const [quarter, setQuarter] = useState(localQuarter || quarters[0].yyyyq);
+	const [subject, setSubject] = useState(localSubject || firstDepartment);
+	const [level, setLevel] = useState(localLevel || "U");
+	const [errorNotified, setErrorNotified] = useState(false);
+
+	const { data: subjects, error: errorGettingSubjects } = useSWR(
+		"/api/public/subjects",
+		fetchSubjectAreas,
+		{
+			initialData: allTheSubjects,
+			revalidateOnMount: true,
+		}
+	);
```

Comment:
This could be a `useBackend` call instead.   Use the techniques you learned on `team03`